dist/stat_data.py

```diff
@@ -288,11 +288,14 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_hazard_max', 'hazard pointer maximum array length', 'max_aggregate,no_scale'),
     CacheStat('cache_hazard_walks', 'hazard pointer check entries walked'),
     CacheStat('cache_hs_insert', 'history store table insert calls'),
+    CacheStat('cache_hs_key_truncate_onpage_removal', 'history store key truncation due to the key being removed from the page'),
```

Comment:
Why `truncate`, I feel it may be confusing. Why can't we use the `remove`?

Reply:
Because we're not removing a key, we're truncating a range of history store keys for that data file key. In that case, what would you call `cache_hs_remove_key_truncate`?

Reply:
I understood the reason behind the `truncate`. As we are removing many keys in the history store as part of the data page key remove. So `truncate` is a better word.